packages/newspringchurchapp/src/prayer/index.js

```diff
@@ -1,343 +1,32 @@
-  alignItems: 'center',
-  marginTop: theme.sizing.baseUnit,
-  marginBottom: theme.sizing.baseUnit,
-}))(View);
-
-const StyledAddPrayerContainer = styled(({ theme }) => ({
-  marginTop: theme.sizing.baseUnit,
-}))(View);
-
-const Tab = ({ index, showAddPrayerCard }) => {
-  const data = prayerMenuData[index - 1];
-  return (
-    <>
-      <StyledPaddedView>
-        <BodyText>{data.description}</BodyText>
-      </StyledPaddedView>
-      {!showAddPrayerCard ? <StyledView>{data.component}</StyledView> : null}
-    </>
-  );
-};
-
-Tab.propTypes = {
-  index: PropTypes.number,
-  showAddPrayerCard: PropTypes.bool,
-};
-
-class PrayerMenu extends PureComponent {
+class PrayerMenuConnected extends PureComponent {
   static navigationOptions = () => ({
     title: 'Prayer',
     header: null,
   });
 
-  static propTypes = {
-    tint: PropTypes.string,
-  };
-
-  state = {
-    index: 0,
-    routes: [
-      {
-        title: 'My Saved Prayers',
-        key: 'saved',
-      },
-      {
-        title: 'My Church',
-        key: 'church',
-      },
-      {
-        title: 'My Campus',
-        key: 'campus',
-      },
-      {
-        title: 'My Community',
-        key: 'community',
-      },
-      {
-        title: 'My Prayers',
-        key: 'prayers',
-      },
-    ],
-    prayerMenuItemSelected: 1,
-    showAddPrayerCard: true,
-    animatedValue: new Animated.Value(0),
-  };
-
-  animate = (toValue) =>
-    Animated.spring(this.state.animatedValue, {
-      toValue,
-      useNativeDriver: true,
-      tension: 30,
-      friction: 7,
-    }).start();
-
-  tabRoute = (index) => () => (
-    <Tab index={index} showAddPrayerCard={this.state.showAddPrayerCard} />
-  );
-
-  handleIndexChange = (index) => this.setState({ index });
-
-  renderTabBar = (props) => (
-    <StyledFeed
-      content={prayerMenuData}
-      renderItem={({ item }) => (
-        <TouchableScale
-          key={item.key}
-          onPress={() => {
-            if (this.state.showAddPrayerCard) {
-              this.animate(1);
-
-              this.setState({
-                showAddPrayerCard: false,
-              });
-            }
-
-            this.setState({ prayerMenuItemSelected: item.key });
-
-            props.jumpTo(item.key);
-          }}
-        >
-          <PrayerMenuCard
-            image={item.image}
-            overlayColor={item.overlayColor}
-            title={item.title}
-            selected={this.state.prayerMenuItemSelected === item.key}
-          />
-        </TouchableScale>
-      )}
-      loadingStateObject={loadingStateObject}
-    />
-  );
-
   render() {
     return (
-      <Animated.View
-        style={{
-          transform: [
-            {
-              translateY: this.state.animatedValue.interpolate({
-                inputRange: [0, 1],
-                outputRange: [0, -(350 * this.state.showAddPrayerCard)],
-              }),
-            },
-          ],
+      <Query query={PRAYER_MENU_CATEGORIES}>
```

Comment:
@redreceipt should we wait for a naming convention change like this until we do the whole Issue to change the convention?

Reply:
if this one gets the go ahead, I'll go ahead and change all rest ;)